src/pages/EntityDetails/Model/Model.js

```diff
@@ -153,17 +169,8 @@ const Model = () => {
     () => generateAppOffersRows(modelStatusData, panelRowClick, query),
     [modelStatusData, panelRowClick, query]
   );
-  const cloudProvider = modelStatusData
-    ? extractCloudName(modelStatusData.model["cloud-tag"])
-    : "";
-
-  const ModelEntityData = {
-    controller: modelStatusData?.model.type,
-    "Cloud/Region": `${cloudProvider} / ${modelStatusData?.model.region}`,
-    version: modelStatusData?.model.version,
-    sla: modelStatusData?.model.sla,
-    provider: modelStatusData?.info?.["provider-type"],
```

Comment:
It seems like this is no longer being represented in the model details page. Is that intentional?